Makefile

```diff
@@ -231,5 +237,29 @@ upload-images: images
 version:
 	@echo ${VERSION}
 
+.PHONY: build-cross
+build-cross: LDFLAGS += -extldflags "-static"
+build-cross:
```

Comment:
Doesn't need to be a part of this change, but I'm wondering if we should consider having "make dep" be required task for some of these, or at least a check if vendor exists and just do it if not.  Again, not an issue for this change but this is a good opportunity to capture the idea.

Reply:
k added dependencies to the build-cross and dist target